src/pipdeptree/__init__.py

```diff
@@ -61,15 +60,13 @@ def guess_version(pkg_key, default="?"):
 
     """
     try:
-        m = import_module(pkg_key)
+        if sys.version_info >= (3, 8):
```

Comment:
Add the covdefaults marker to allow this if/else to be covered correcvtly.

Reply:
Added